src/tablecellproperties/ui/tablecellpropertiesview.js

```diff
@@ -562,7 +595,7 @@ export default class TableCellPropertiesView extends View {
 
 		widthInput.set( {
 			label: t( 'Width' ),
-			class: 'ck-table-cell-properties-form__width',
+			class: 'ck-table-properties-form__width',
```

Comment:
Why this change? It looks like an error. Did we extracted some common styles that I've missed?

Reply:
Yeah, I noticed the duplication and I extracted them in this PR.

Reply:
I went even further in https://github.com/ckeditor/ckeditor5-table/pull/244/commits/dc063334e91bad1d8b99a3817e1bc7caad639a5f.